sycl/doc/GetStartedGuide.md

```diff
@@ -43,6 +43,8 @@ and a wide range of compute accelerators such as GPU and FPGA.
 * `python` - [Download](https://www.python.org/downloads/)
 * `ninja` -
 [Download](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages)
+* `hwloc` version 2.3 or later
```

Comment:
```suggestion
* `hwloc` version 2.3 or later (Linux only)
```

Reply:
@igchor is `hwloc` a linux-only requirement? I recall installing `hwloc` in Windows CI machines...

Reply:
We initially required hwloc on Windows but it turned out we couldn't support dynamic linking due to dependentloadflag flag being used. Since then we changed to linking statically with hwloc on windows (and so, user doesn't have to install anything now).